package.json

```diff
@@ -49,10 +49,22 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "eb:prod": "npm run build && npm run start:prod",
-    "prettier": "prettier --write \"./src/**/*.{js,jsx}\""
+    "prettier": "prettier --write \"./src/**/*.{js,jsx}\"",
+    "storybook": "start-storybook -p 6006 -s public",
```

Comment:
Can you remove `public` ? we dont want it to build to the public folder but rather for Storybook folder `storybook-static`. This will allow us to coordinate 2 different deployments. The public folder should only be for the react-create-app application